python/cudf/cudf/_lib/cpp/io/functions.pxd

```diff
@@ -161,6 +161,33 @@ cdef extern from "cudf/io/functions.hpp" \
     cdef unique_ptr[vector[uint8_t]] \
         write_parquet(write_parquet_args args) except +
 
+    cdef cppclass write_parquet_chunked_args:
+        cudf_io_types.sink_info sink
+        cudf_io_types.compression_type compression
+        cudf_io_types.statistics_freq stats_level
+        const cudf_io_types.table_metadata *metadata
+
+        write_parquet_chunked_args() except +
+        write_parquet_chunked_args(cudf_io_types.sink_info sink_,
+                                   cudf_io_types.table_metadata *table_metadata_,
+                                   cudf_io_types.compression_type compression_,
+                                   cudf_io_types.statistics_freq stats_lvl_) except +
+
+    cdef shared_ptr[pq_chunked_state] \
+        write_parquet_chunked_begin(write_parquet_chunked_args args) except +
+
+    cdef void write_parquet_chunked(cudf_table_view.table_view table_,
+                                    shared_ptr[pq_chunked_state]) except +
+
+    cdef void write_parquet_chunked_end(shared_ptr[pq_chunked_state])
```

Comment:
Can this API throw? Should likely annotate to catch exceptions:
```suggestion
    cdef void write_parquet_chunked_end(shared_ptr[pq_chunked_state]) except +
```

Reply:
Added  ```except +```  in latest commit